src/pages/home.tsx

```diff
@@ -28,21 +28,16 @@ export default function Index() {
   function handleClick(e: number) {
     if (Number(e) === 0) {
       router.push('/home');
-      return
+      return;
     } else {
       setGroup(e);
       router.push({ pathname: '/home', query: { group_id: e } });
     }
```

Comment:
Quando tem o return o **else** é desnecessário
```suggestion
    } 
    setGroup(e);
    router.push({ pathname: '/home', query: { group_id: e } });
```